src/components/Footer/index.js

```diff
@@ -47,46 +27,29 @@ const useStyles = makeStyles(theme => ({
     [theme.breakpoints.up('md')]: {
       justifyContent: 'flex-end'
     }
-  },
-  community: {
-    width: '50%',
-    [theme.breakpoints.up('sm')]: {
-      width: 'auto'
-    }
-  },
-  project: {
-    width: '50%',
-    [theme.breakpoints.up('sm')]: {
-      width: 'auto'
-    }
   }
 }));
 
 function Footer(props) {
   const classes = useStyles(props);
 
   return (
-    <Grid container className={classes.root}>
-      <Grid
-        item
-        xs={12}
-        container
-        alignItems="flex-start"
-        className={classes.layout}
-      >
-        <Grid item xs={12} sm={7} className={classes.about}>
-          <About />
-        </Grid>
-        <Grid item xs={12} sm={5} container className={classes.organisation}>
-          <Grid item className={classes.community}>
+    <div className={classes.root}>
+      <Container>
+        <Grid item xs={12} container spacing={3} alignItems="flex-start">
```

Comment:
@DavidLemayian  unequal margin on Grid makes the content align more to the left on mobile, `spacing={3}` is the cause
![Screenshot from 2019-10-16 09-29-53](https://user-images.githubusercontent.com/7962097/66893941-e33e9200-eff7-11e9-9ad4-7f54c8c60d14.png)

Reply:
So yeah, had seen this and couldn't figure it out. A little coffee and a lot of Googling later right now has shown a couple of things - "gutters" are an inspiration from Bootstrap and are a best practice so technically it's working as expected... Buuut... It seem that because we're also passing `xs={12}` prop, it messes with everything.

I've gone ahead and removed `xs={12}` because by default, it seems passing `container` prop on `<Grid/>` gives it full width.

Reply:
@DavidLemayian :100: